packages/react/src/editor/BlockNoteDefaultUI.tsx

```diff
@@ -48,11 +49,11 @@ export function BlockNoteDefaultUI(props: BlockNoteDefaultUIProps) {
       {editor.tableHandles && props.tableHandles !== false && (
         <TableHandlesController />
       )}
-      {editor.comments && (
-        <>
-          <FloatingComposerController />
-          {props.comments !== false && <FloatingThreadController />}
-        </>
+      {editor.comments && props.commentsComposer !== false && (
+        <FloatingComposerController />
```

Comment:
I think both elements could be part of the same `comments` flag, wdyt? (i.e.: if you opt out if this you need to render all comment-releated elements yourself)

Reply:
I think since the `ThreadsSidebar` is part of the API, I think it's gonna be annoying if people have to re-add the `FloatingComposerController` when they use it. Also this is a bit of a different situation vs the other flags, but for those it's the case that 1 flag is for 1 UI element, whereas here the `comments` flag would control multiple elements.

Reply:
then we should also name the other one `floatingThreads` or sth to be more descriptive?

imo a single `comments` flag is easiest atm